regression/arrays/double_index/main.c

```diff
@@ -0,0 +1,21 @@
+extern void __VERIFIER_error() __attribute__ ((__noreturn__));
```

Comment:
Are these examples taken from SV-COMP? If so, we should also add the appropriate license header.

Reply:
Not this one but some of them are. Added license headers.